src/styles/app.css

```diff
@@ -0,0 +1,10 @@
+.App{
+    display: flex;
+    flex-direction: column;
+}
+
+.nasa-logo{
+    margin: auto;
+    width: 400px;
+    height: 200px;
+}
```

Comment:
There should be a space before the opening bracket in each class, and should also be using 2 spaces of indentation as with the rest of the project
```
.App {
  display: flex;
  flex-direction: column;
}

.nasa-logo {
  margin: 0 auto;
  width: 400px;
  height: 200px;
}
```

Also, the margin property does not work with the auto keyword for vertical directions, what youre effectively saying here is
```
margin-top: auto;
margin-right: auto;
margin-bottom: auto;
margin-right: auto;
```
This should be changed for 
```
margin: 0 auto;
```